icu4c/source/tools/toolutil/writesrc.h

```diff
@@ -95,13 +106,14 @@ usrc_writeUCPTrieStruct(FILE *f,
                         const char *prefix,
                         const UCPTrie *pTrie,
                         const char *indexName, const char *dataName,
-                        const char *postfix);
+                        const char *postfix,
+                        UTargetSyntax syntax);
 
 /**
- * Writes the UCPTrie arrays and struct values.
+ * Writes the UCPTrie arrays and struct values as C code.
```

Comment:
Why "as C code" when you are parameterizing it?

Reply:
I was going to add a separate function but I added a parameter instead. Fixed